Lib/inspect.py

```diff
@@ -2971,6 +2973,18 @@ def __init__(self, parameters=None, *, return_annotation=_empty,
                             # There is a default for this parameter.
                             seen_default = True
 
+                    if kind == _VAR_POSITIONAL:
+                        var_positional_count += 1
+                        if var_positional_count > 1:
```

Comment:
It seems, ``*_count`` actually didn't count anything, isn't?  I suggest use bool values instead and rename variables to something like ``seen_var_positional``.